p2p/service.py

```diff
@@ -39,7 +39,7 @@ class BaseService(ABC, CancellableMixin):
     _tasks: 'WeakSet[asyncio.Future[Any]]'
     _finished_callbacks: List[Callable[['BaseService'], None]]
     # Number of seconds cancel() will wait for run() to finish.
-    _wait_until_finished_timeout = 5
+    _wait_until_finished_timeout = 10
```

Comment:
Hmm, we shouldn't have to wait this long... even 5s is already too much, really. If we're seeing timeouts when terminating we must have something that is failing to cleanup

Reply:
Didn't mean to include this change, reverted.